test/sslapitest.c

```diff
@@ -4487,19 +4487,19 @@ static int test_ciphersuite_change(void)
  * Test 12 = Test all ECDHE with TLSv1.2 client and server
  * Test 13 = Test all FFDHE with TLSv1.2 client and server
  */
+# ifndef OPENSSL_NO_EC
+static int ecdhe_kexch_groups[] = {NID_X9_62_prime256v1, NID_secp384r1,
```

Comment:
nit: `static const int ...`

Reply:
Thanks; fixup pushed.

Reply:
The CI is unhappy about this, though, since we assign these to an `int *`, and that variable sometimes also points to a stack-allocated `int` (that is mutable in the tests).  I reverted the fixup for now, as propagating const everywhere would induce a bit of churn in the test body itself.